test/python/test_qcvv_fitters.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=missing-docstring
+
+import numpy as np
+
+from qiskit.tools.qcvv import fitters
+from .common import QiskitTestCase
+
+
+class TestQCVVFitters(QiskitTestCase):
+
+    def test_exp_fit_fun(self):
+        a_in = 2
+        x_in = 3
+        tau_in = 4
+        c_in = 1
+        result = fitters.exp_fit_fun(x_in, a_in, tau_in, c_in)
+        self.assertEqual(1.9447331054820294, result)
```

Comment:
Non-blocking: have you tried using [`self.assertAlmostEqual`](https://docs.python.org/3.7/library/unittest.html#unittest.TestCase.assertAlmostEqual)?

Reply:
This should be changed as it is never good to do an equality check for floats.